### PR TITLE
In this branch we: Create the Awesome books: plain JavaScript with objects project

### PR DESCRIPTION
**Pull request by MohammedElseddik**
- Create index.html.
- Create main.js
- Create a javascript object to have the books.
- Display the books list from the javascript object.
- Save the javascript object to local storage and retrieve it.
- Add new books functionality.
- Add remove book functionality.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    bookListUi.appendChild(listUi);
+  });
+};
+
+function setLocalStorage() {
+  localStorage.setItem('BookList', JSON.stringify(arrayBooks));
+}
+
+// Create the Books array
+const addBookHandler = () => {
```
Comment (micheaol) [inline app.js:41]: Please ensure that the app performs as expected, Please find below to see the error received when I tried to add a book.
Kindly fix it:
image

As a result, I am unable to check the remove book function

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @MohammedElseddik , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.js (hunk 1 above)
Comment (MohammedElseddik) [reply app.js:41]: @micheaol the app works perfectly with the live server could you please give me any help about this issue?

Comment (micheaol) [reply app.js:41]: @MohammedElseddik ,

I will suggest that you should try to use `condole.log` before pushing. With this you will know the result of the object

Comment (MohammedElseddik) [reply app.js:41]: @micheaol I used console.log to check before but  as I told you when I open the app with the Live Server everything work perfectly but when I do not open it with the live server this problem appears

Comment (MohammedElseddik) [reply app.js:41]: @micheaol  could you please check it again for me I think I fix it before ask for another review?

Comment (micheaol) [reply app.js:41]: @MohammedElseddik ,
Please give a sec

Comment (micheaol) [reply app.js:41]: @MohammedElseddik ,

I still received the same error message, maybe you should take your time to check online for a solution and I will do the same to look up a solution if I can find any.

Comment (MohammedElseddik) [reply app.js:41]: @micheaol  Thank you so much I fix it you can check it it is on line 70  app.js file

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**